features/step_definitions/files_steps.rb

```diff
@@ -30,3 +30,10 @@
   expect(uncommitted_files).to eql [file_name]
   expect(IO.read file_name).to eql content
 end
+
+
+Then(/^my uncommitted file "(.+?)" is still stashed away$/) do |file_name|
+  expect(uncommitted_files).to_not include file_name
```

Comment:
That's a great idea! `git stash show` doesn't show uncommitted files, unfortunately, but I'll think about a way to make this happen. Maybe creating a new branch with the stashed changes (`git stash branch`), checking it, then throwing it away.